Doc/library/calendar.rst

```diff
@@ -586,10 +586,16 @@ The following options are accepted:
    or as an HTML document.
 
 
+.. option:: --firstweekday WEEKDAY, -f WEEKDAY
+
+   The weekday to start each week.
+   Must be a number between 0 and 6.
+   Defaults to 0.
```

Comment:
Please clarify that 0 is Monday and 6 is Sunday. Also add this in the option help.

Reply:
I made changes based on your feedback.